EE_API_automation/pytest/src/support/planner_jsons/edit_wi_state.json

```diff
@@ -3,7 +3,6 @@
     "id": "$wi_id",
     "attributes": {
       "system.state": "$state",
-      "system.description.markup": "Markdown",
```

Comment:
Why was this removed?

Reply:
The Markdown field in the request is optional while updating State. Its already tested as part of PATCH description test.